test/backup/backup_test.go

```diff
@@ -901,6 +903,35 @@ func TestBackups_integration(t *testing.T) {
 			assert.Contains(t, err.Error(), "CPUPercentage")
 		})
 	})
+
+	t.Run("cancel backup", func(t *testing.T) {
+		testsuit.CreateTestSchemaAndData(t, client)
+		defer testsuit.CleanUpWeaviate(t, client)
+
+		class := "Pizza"
+		backend := backup.BACKEND_FILESYSTEM
+		id := fmt.Sprint(random.Int63())
+		ctx := context.Background()
+
+		// Arrange: Check fixture data exists and create a backup of it
```

Comment:
am not the biggest fan of these standard comments 😅  - for me they make sense for bigger tests but create noise for these small ones